src/common/integration_test_base_class.py

```diff
@@ -0,0 +1,79 @@
+# standard library
+import unittest
+
+# third party
+import mysql.connector
+
+# first party
+from delphi.epidata.client.delphi_epidata import Epidata
+from delphi.epidata.server._limiter import limiter
+
+
+class BasicIntegrationTest(unittest.TestCase):
+    """Basic integration test class"""
+
+    def __init__(self, methodName: str = "runTest") -> None:
+        super().__init__(methodName)
+        self.delete_from_tables_list = []
+        self.truncate_tables_list = []
```

Comment:
do we need both `DELETE` and `TRUNCATE`?  `TRUNCATE` is faster for bigger tables, but that shouldnt really matter in integration tests.  `TRUNCATE` will also reset `AUTO_INCREMENT` counters, but hopefully all of our tests are agnostic to any assigned `id` values and such.

it will be simpler to have only one of these, but if there is a good reason for having both, thats fine too.

Reply:
That is unfortunate but yes, we need both of them. There are some tables with relations that can not be truncated(because of that relations) so the only way to clear them is to use `DELETE FROM <table>` statement.